lamindb/_file.py

```diff
@@ -387,7 +386,7 @@ def log_storage_hint(
     if check_path_in_storage:
         display_root = storage.root  # type: ignore
         # check whether path is local
-        if not storage.root.startswith(("s3://", "gs://")):  # type: ignore
+        if fsspec.utils.get_protocol(storage.root) == "file":  # type: ignore
```

Comment:
Why don't you use the `isinstance(filepath, LocalPathClasses)` check here?

Reply:
Here `storage` is a model with `root` being a string.

Reply:
Yes, you're right! 

To make this more readable (`fsspec.utils.get_protocol(storage.root) == "file"` isn't understandable at all), could we introduce a helper function?

```
def is_local_path(path: [str, UPath, Path]) -> bool:
    if isinstance(path, str):
         return fsspec.utils.get_protocol(storage.root) == "file"
    else:
         return isinstance(path, LocalPathClasses)
```

This could be imported like so:
```
from lamindb.dev.upath import is_local_path, is_cloud_path
```
And then be used anywhere.

(Not sure whether the `_path` is needed.)